integrationtests/src/test/java/io/kroxylicious/proxy/KrpcFilterIT.java

```diff
@@ -298,6 +301,92 @@ public void shouldModifyFetchMessage() throws Exception {
 
     }
 
+    @Test
+    public void proxySslClusterPlain() throws Exception {
+        String proxyAddress = "localhost:9192";
+
+        var certificateGenerator = new KeytoolCertificateGenerator();
+        certificateGenerator.generateSelfSignedCertificateEntry("test@redhat.com", "localhost", "KI", "RedHat", null, null, "US");
+
+        try (var cluster = KafkaClusterFactory.create(KafkaClusterConfig.builder().testInfo(testInfo).build())) {
+            cluster.start();
+            try (var admin = Admin.create(Map.of(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, cluster.getBootstrapServers()))) {
+                admin.createTopics(List.of(
+                        new NewTopic(TOPIC_1, 1, (short) 1),
+                        new NewTopic(TOPIC_2, 1, (short) 1))).all().get();
+            }
+
+            String config = """
+                    proxy:
+                      address: %s
+                      keyStoreFile: %s
+                      keyPassword: %s
+                    clusters:
+                      demo:
+                        bootstrap_servers: %s
+                    filters:
+                    - type: ApiVersions
+                    - type: BrokerAddress
+                    - type: FetchResponseTransformation
```

Comment:
Could we remove the fetch response transform as the test is about SSL? Seeing ciphertext in the input vs plaintext in the assertions is distracting. Or did you want to check that SSL and filtering are working together?

Reply:
You are right, the `FetchResponseTransformation` is really nothing to do with this test.  The integration test really just needs to demonstrate that the connection works and that TLS is actually used.  I'll improve the test.